README.md

```diff
@@ -1,66 +1,47 @@
-# FEE YEAR-AP: A Social Media Analytics Platform
+# Jellyfish: A Social Media Analytics Platform
 
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](code_of_conduct.md)
 [![license](https://img.shields.io/badge/license-Apache%202-blue)](License.txt)
+
 # Introduction
 
 ## What is this repo?
 
-This repo contains the main code of the "FEE YEAR Analytics Platform".
+This repo contains the main code of the "Jellyfish Analytics Platform".
+
+## What is the Jellyfish
 
-## What is the FEE YEAR-AP
+It is an OS application that will collect analytics data from YouTube, Google Ads
+and Facebook and generate reports about it.
 
-It is an application that will collect analytics data from YouTube, Google Ads
-and Facebook and generate reports about it, so that FEE can improve their
-reach on the population and spread their ideas on liberty.
+The idea on making Jellyfish possible came by [FEE](https://fee.org/about)
 
 # Building and Running
 
 The AP is composed of the following subsystems:
   - Console App
```

Comment:
AP here is from the Application Platform?  And we only have one subsystem now. Maybe, we should remove these lines.

Reply:
AP for analytics platform. You're right, I'll just remove this part.